test/features/ui_authorization_auth_code.feature

```diff
@@ -0,0 +1,46 @@
+@method=POST @endpoint=/authorization/auth-code
+Feature: The API sends the accepted consent and permitted scopes and retrieves the authorization code.
+
+  This endpoint is invoked from JS application to send the accepted consent and permitted scopes.
+
+  @smoke @unit @positive
+  Scenario: Successfully retrieve the authorization code
+
+    Given The end-user wants to send the accepted consent and permitted scopes via JS application
+    When POST request with given current date as requestTime "jskSD23wes324545F" as transactionId "string" as permittedAuthorizeScopes "string" as acceptedClaims is sent
+    Then The response is received
+    And The response should be returned in a timely manner
+    And The response should have status 200
+    And The response header content-type should be "application/json; charset=utf-8"
+    And The response should contain authorization code
+    And The response should match json schema
+
+  @unit @negative
+  Scenario Outline: Unable to retrieve the authorization code because of an invalid transactionId parameter
+
+    Given The end-user wants to send the accepted consent and permitted scopes via JS application
+    When POST request with given current date as requestTime "<transactionId>" as transactionId "<permittedAuthorizeScopes>" as permittedAuthorizeScopes "<acceptedClaims>" as acceptedClaims is sent
+    Then The response is received
+    And The response should be returned in a timely manner
+    And The response should have status 200
+    And The response header content-type should be "application/json; charset=utf-8"
+    And The response should match json schema
```

Comment:
Can we also have a condition to check for specific error code like "invalid_transaction_id" ?